studia-historica-upsaliensa.csl

```diff
@@ -0,0 +1,588 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="note" version="1.0" and="symbol" initialize="false" demote-non-dropping-particle="never" xmlns="http://purl.org/net/xbiblio/csl">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Studia Historica Upsaliensa </title>
+    <title-short>Studia</title-short>
+    <id>http://www.zotero.org/styles/studia-historica-upsaliensa</id>
+    <link href="http://www.zotero.org/styles/studia-historica-upsaliensa" rel="self"/>
+    <link href="http://www.zotero.org/styles/chicago-fullnote-bibliography" rel="template"/>
+    <link href="http://www.hist.uu.se" rel="documentation"/>
```

Comment:
Is there an actual (online or offline) document that describes this citation format? Giving the URL of the department helps, but a direct link to the style guide would be more useful.